frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetailsPage.tsx

```diff
@@ -92,13 +103,7 @@ const BareMetalHostDetailsPage: React.FC<BareMetalHostDetailsPageProps> = (props
       ]}
       kind={referenceForModel(BareMetalHostModel)}
       resources={resources}
-      menuActions={menuActionsCreator}
-      customData={{
-        hasNodeMaintenanceCapability: !!maintenanceModel,
-        maintenanceModel,
-        bmoEnabled,
-        t,
-      }}
+      customActionMenu={customActionMenu}
```

Comment:
Robb figured out that we can also do:
```suggestion
      customActionMenu={(kindObj: K8sModel, obj: K8sResourceKind) => (
	        <LazyActionMenu context={{ [referenceForModel(BareMetalHostModel)]: obj }} variant={ActionMenuVariant.DROPDOWN} />
	    )}
```
and remove the `customActionMenu` above.